package.json

```diff
@@ -113,8 +115,8 @@
     "cross-env": "^5.1.4",
     "doctoc": "^1.3.0",
     "doctrine": "^2.0.0",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.0.1",
+    "enzyme": "^3.7.0",
```

Comment:
I guess these changes should be part of https://github.com/stardust-ui/react/pull/493

Reply:
Of #492 👍 I will rebase this PR after it will be merged, it's in TODO 😺